test/screenshot/golden.json

```diff
@@ -1,25 +1,23 @@
-  "floating-label/index.html": "cca310cdd33bad5200c1ec201a8d07ffb327b64ac49db525979a1ab1ac75ca8b",
-  "notched-outline/index.html": "73394b6d7dcd94bc930dc740db8199000b94d365d7d703381ef01b3318b97d8c",
-  "text-field/icon/index.html": "8a1d1969d6e7df5ea9def6ad1eb1734d5d3486c6bdda6c7721c494d8c46d37c3",
-  "text-field/helper-text/index.html": "c246d6f593fe01901d26973d7c7384232892cc44aada19e82c8709b82f1a20e1",
-  "text-field/index.html": "5d753f3ebcc1ccfebf07057b87e579fa21448b0bb98329212d8420bcae21330e",
-  "top-app-bar/fixed.html": "90534d59d40f8c050aae022e94b0afa022a00d1e00c19e3f92dcc1908efcd831",
-  "chips/index.html": "949725ee6658e0c4edcdecd2bf057fda5fb814473f7a527da5c96e6392b84015",
-  "select/index.html": "5ce288649fe5e78d21ab9315a35cc2a22504176b8abaf56b92579121df51f073",
-  "tab-indicator/index.html": "19f1befc0947128b46d84e1fdee73741bffcd63950bf0cddaadd9ae283ab85f8",
-  "tab-scroller/index.html": "726a001f1178c06e3fad4a943c6253c8c89084e3867bced469be5bb5b8529278",
-  "tab/index.html": "2151d184cc420fdf19c3e9b6e8c553ee1cbfc7df4ebbda73b7a669dfca939bef"
+  "button": "b01eeaad126491150baf47a736f5d5fbcfc284e62a47d91e9e50dcf1163925c1",
+  "card": "b2fd82763c383be438ff6578083bf9009711c7470333d07eb916ab690fc42d31",
+  "line-ripple": "56b136db2dc7e09260849447e6bde9b55a837af332a05d9f52506ab1c95e2e57",
+  "fab": "db36f52195c420062d91dd5ebe5432ad87247b3c1146fd547b0a195079bbce2f",
+  "floating-label": "eb625c092696d98c5acfc08e8cf6fdbf2699cb211363da0f98931fecad4d3d65",
+  "material-icon": "442b39fb22d2c7a74efb23ca098429b471501ce21df8662327bbf9871fe0bcb0",
+  "notched-outline": "a295f6038913e65b2759c1333494f9e3b410e33b139d1cf3edfd19c9145eb0ee",
+  "select": "f91c1ef2df6b06568ee28e1c94d1238944031c03163546eb785c9950f7381c89",
+  "tab": "f9fba67bb0aa416e3b82da4269114a4fee1c342d7b362c5e1b177e6cbd6dc2e1",
+  "tab-indicator": "7ce7ce8fd50301c67d7ebfb0ba953208260ce2881bee0c7e640c46bf60dc90b6",
+  "tab-scroller": "468866dd0c222b36b55485ab44a5760133a4ddfb2a6cf81e6ae4672d7e02a447",
+  "text-field": "cc603806aba9774eafa53001478f55be32540840abfc683fb32aa3debf3ee86d",
+  "text-field/helper-text": "dd523d5b6381a14be40f9c4994889ca908d317f7072b85b68be0c91b8b664b97",
+  "text-field/icon": "c34dae5444deec222533b3f43448c0393b0c8543a5af4e50cc12d71611f366a7",
+  "top-app-bar/fixed": "90534d59d40f8c050aae022e94b0afa022a00d1e00c19e3f92dcc1908efcd831",
+  "top-app-bar/prominent": "34d03c5809067cc06a67dce1e03d3e1edaf306360479ac1955299d3eeffc6113",
+  "top-app-bar/short": "fe50f5577ef8c9def95450090eae4033192034dd0e0890407d4ab80e04e3e6a1",
+  "top-app-bar/shortCollapsed": "3565597194402f65bfc48edcea298a49c890c70100befe3ae19dd3cf24f17716",
+  "top-app-bar/standard": "90534d59d40f8c050aae022e94b0afa022a00d1e00c19e3f92dcc1908efcd831",
+  "top-app-bar/standardNoActionItems": "9102ece0efc0a040e0dc2b097c66d2ee5ecc9c91759aeb16ea5aa2baabe7307a",
+  "top-app-bar/standardWithNavigationIconElement": "91b10df9c4faf85ba63c84b1cf82a1c1143a3d8c4ba7ed8af477fabdc65ec25e"
```

Comment:
this is shorter because chips and material-icon/menu are not in the list. Material-icon/menu is an old test which was removed. Chips not in yet.